playbooks/common/openshift-node/config.yml

```diff
@@ -160,6 +160,7 @@
     openshift_node_master_api_url: "{{ hostvars[groups.oo_first_master.0].openshift.master.api_url }}"
     etcd_urls: "{{ hostvars[groups.oo_first_master.0].openshift.master.etcd_urls }}"
     embedded_etcd: "{{ hostvars[groups.oo_first_master.0].openshift.master.embedded_etcd }}"
+    first_master_ip: "{{ hostvars[groups.oo_first_master.0].openshift.common.ip }}"
```

Comment:
we'll want to prefix this var for the openshift_node role...  old style was to use openshift_node_<var_name>, newer style started using osn_<var_name>...  It seems that openshift_node_<var_name> is more consistent with galaxy recommendations though...

On an aside, I think variables above (embedded_etcd, etcd_urls) is used when configuring flannel on nodes, and should probably have a prefix as well.